mcux/mcux-sdk/drivers/edma_rev2/fsl_edma_rev2.c

```diff
@@ -237,6 +240,11 @@ status_t EDMA_ConfigureTransfer(edma_config_t *cfg, int channel,
         break;
     }
 
+#if defined FSL_FEATURE_MEMORY_HAS_ADDRESS_OFFSET && FSL_FEATURE_MEMORY_HAS_ADDRESS_OFFSET
+    saddr = MEMORY_ConvertMemoryMapAddress((uint32_t)(saddr), kMEMORY_Local2DMA);
```

Comment:
might be useful to add a comment here stating that translation will only be performed if saddr/daddr are from DTCM